pages/linux/grubby.md

```diff
@@ -0,0 +1,16 @@
+# grubby
+
+> Command line tool for configuring grub and zipl.
+> More information: <https://src.fedoraproject.org/rpms/grubby>.
```

Comment:
```suggestion
> More information: <https://manned.org/man/grubby.8>.
```
We generally prefer linking to the man page/usage documentation (if it is present then we reference the VCS repository.)